certified-connectors/ZohoSign/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "<Enter Your Client ID>",
+          "scopes": [
+            "ZohoSign.documents.all",
+            "ZohoSign.account.all",
+            "ZohoSign.templates.all"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://accounts.zoho.com/oauth/v2/auth"
+            },
+            "tokenUrl": {
+              "value": "https://accounts.zoho.com/oauth/v2/token"
+            },
+            "refreshUrl": {
+              "value": "https://accounts.zoho.com/oauth/v2/auth"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Zoho Corporation"
```

Comment:
@ZohoSign-integ It is a syntax error, Please add a comma an the end of property

Reply:
@ZohoSign-integ Please fix it

Reply:
@sriyen-msft fixed it. Please check this.